### ISSUE TITLE
Problems with shutting down Fury

### ISSUE THREAD

**Issue by odisseus** [propensive/fury-old#1049]
1. The Nailgun server does not terminate after Fury is stopped
    1. The threads that keep it alive seem to be related to the Bloop launcher
2. The Bloop server sometimes doesn't terminate even after Nailgun is killed
3. Commands `pkill -f fury` and `killall fury` should not be used in the launcher script, because they also kill the script
4. The command `pgrep bloop` fails to find the Bloop process on some machines
    1. This seems to be related to the length of the command (which is extremely long in case of Bloop)

**Comment by propensive** [comment]
I agree. Are the first two issues caused by the third? We also shouldn't use `killall` because it's not standard on every machine.
